src/index.js

```diff
@@ -58,30 +58,37 @@ async function getReviewers(username) {
 
   // 先抽 mentor
   if (mentor) {
-    core.info(`Requesting mentor: ${mentor}`);
-    reviewers.push(mentor);
+    if (initialReviewers.includes(mentor)) {
+      core.info(`Mentor ${mentor} already requested.`);
+    } else {
+      core.info(`Requesting mentor: ${mentor}`);
+      reviewers.push(mentor);
+    }
   } else {
     core.info(`No mentor found.`);
   }
 
   // 再從同專案團隊優先抽滿
-  const belongingTeamMembers = Object.values(config.teams)
-    .find(teamMembers => teamMembers.includes(username))
-    .filter(member => member !== username && member!== mentor);
+  {
+    const belongingTeamMembers = Object.values(config.teams)
+      .find(teamMembers => teamMembers.includes(username))
+      .filter(member => member !== username);
 
-  const remainingCount = targetCount - reviewers.length;
-  const teamReviewers = randomPick(remainingCount, belongingTeamMembers);
-  reviewers.push(...teamReviewers);
+    const candidates = _.difference(belongingTeamMembers, reviewers, initialReviewers);
+    const remainingCount = targetCount - reviewers.length;
+    const teamReviewers = randomPick(remainingCount, candidates);
+    reviewers.push(...teamReviewers);
 
-  core.info(`Requesting team members: ${teamReviewers}`);
+    core.info(`Requesting team members: ${teamReviewers}`);
+  }
 
   // 不夠的話從 mentorship group 抽到滿
   if (reviewers.length < 2) {
```

Comment:
這裡是不是要換成 targetCount ?
```suggestion
  if (reviewers.length < targetCount) {
```

Reply:
要 可惡怎麼沒改到